src/components/features/NewFurniture/NewFurniture.js

```diff
@@ -105,7 +124,10 @@ class NewFurniture extends React.Component {
             </div>
             <div className={`row ${styles.productsRow} ${this.state.className}`}>
               {categoryProducts
-                .slice(activePage * 8, (activePage + 1) * 8)
+                .slice(
+                  activePage * this.rwdCardsInRow[rwdMode.mode],
```

Comment:
Moim zdaniem możnaby zrobić zmienną do której przypisałoby się wartość this.rwdCardsInRow[rwdMode.mode]
zwiększyłoby to czytelność kodu

Reply:
Dzięki za uwagę, Myślałem o zmiennej ale ostatecznie zostałem przy takim generycznym zapisie, łatwo jest od razu dojść do obiektu w którym ustawione są ilości itemów i dokonać ew. zmian. Przy zmiennej byłby jeden więcej do wykonania. Pozostawiam jak jest :)

Reply:
Jasne, zrozumiałe :D